app/views/spree/order_mailer/cancel_email.html.haml

```diff
@@ -1,11 +1,29 @@
-%h3
-  = t(".customer_greeting", name: @order.bill_address.firstname)
-%h4
-  = t(".instructions")
-%span.clear
+%table.social.white-bg{width: "100%"}
+  %table.column{align: "left"}
```

Comment:
Can we set this property in a CSS file?

Reply:
```css
.column {
  width: 300px;
  float: left;

  tr td {
    padding: 15px;
  }
}
```
It's already been set the property to be on left.